libs/common/src/helpers/with-boolean-feature-flag.tests.ts

```diff
@@ -0,0 +1,42 @@
+const originalEnv = { ...process.env };
+
+export const withBooleanFeatureFlagTests = ({ featureFlagName, getFeatureFlagValue }: { featureFlagName: string; getFeatureFlagValue: () => boolean }) => {
```

Comment:
FWIW could use FeatureFlag type from `is-feature-flag-enabled`

```suggestion
export const withBooleanFeatureFlagTests = ({ featureFlagName, getFeatureFlagValue }: { featureFlagName: FeatureFlag; getFeatureFlagValue: () => boolean }) => {
```